README.rst

```diff
@@ -128,60 +128,31 @@ file or disable package repositories update in config.yaml.
 Using the RPMs
 --------------
 
-We decided that's is a little bit intrusive to install all the
-produced RPMs since some of them may not fit your needs. At the end of
```

Comment:
We are recommending only the virtualization packages and the user is executing the install command himself, so he has control over what's installed. I think this sentence is unnecessary.